p2p/p2p.go

```diff
@@ -50,15 +50,15 @@ type Service struct {
 	bootstrapNodes []*ps.PeerInfo
 }
 
-// ServiceConfig is used to initialize a new p2p service
-type ServiceConfig struct {
+// Config is used to configure a p2p service
+type Config struct {
```

Comment:
not sure about this rename..maybe we will have other p2p configs in the future

Reply:
will address later on